website/partners/api/v2/serializers/partner_event.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+
+from partners.models import PartnerEvent
+from thaliawebsite.api.v2.serializers import CleanedHTMLSerializer
+
+
+class PartnerEventSerializer(serializers.ModelSerializer):
+    """Partner events serializer."""
+
+    class Meta:
+        """Meta class for partner events serializer."""
+
+        model = PartnerEvent
+        fields = ("pk", "title", "description", "start", "end", "location", "url")
```

Comment:
This does not include the partner, which I would expect to be given (even though it might be null or a string). It's not a problem per se but may be something to add.

Reply:
Also note that there's no `/partners/:id` endpoint which is also okay but maybe strange.

Reply:
Partner events are not attached to partners. They are only links to the website of their events. So there is nothing more to add. Would probably be more appropriate to call them 'external events' and just move the model to the events package.

Reply:
Still their model does include a partner or 'other partner' string. I do agree moving them to events app would be a good idea.